examples/simple-usage/lifecycle.ts

```diff
@@ -0,0 +1,23 @@
+import { TaskExecutor } from "@golem-sdk/golem-js";
+
+/**
+ * This example shows how to use the TaskExecutor lifecycle events.
+ */
+(async function main() {
+  console.log("Creating TaskExecutor and registering for lifecycle events...");
+  const executor = await TaskExecutor.create("golem/alpine:latest");
+
+  executor.events.on("terminating", () => {
```

Comment:
If we want to notify the user about a condition in which they should shut-down their application, we need to have a "industry standard approach implemented".